torchaudio/models/wav2vec2/model.py

```diff
@@ -967,12 +989,15 @@ def hubert_pretrain_model(
         skip_masked,
         skip_nomask,
     )
-    return HuBERTPretrainModel(
+    model = HuBERTPretrainModel(
```

Comment:
nit: not necessarily in the scope of this PR, but I noticed the docs for the arguments of HuBERTPretrainModel don't match https://github.com/pytorch/audio/pull/2716/files#diff-41adbd739db9af82fe56cc5ad4883f8e89c2667c18b53d487e683d62667fa489R140